docs/platform/what-is-tizen/versions/tizen-5-0-m2.md

```diff
@@ -0,0 +1,42 @@
+# Tizen 5.0
+
+[The second milestone (M2) release of Tizen 5.0](../../release-notes/tizen-5-0-m2.md) was announced in October 2018. Tizen has expanded its horizon from smart devices to IoT devices. Currently, Tizen is used for various kinds of smart devices.
+Tizen 5.0 adds more enhancements to the services and provides the opportunity to develop service-centric applications. With the result, the end-users will get a better experience with more enhanced applications. Tizen 5.0 provides APIs such as Tizen Interface Definition Language (TIDL), RPC-port for easy integration of various kinds of services such as Smart-things. Core open sources such as Blink, GStreamer, EFL, Wayland, and so on are upgraded.
+
+The key features of Tizen 5.5 are as follows:
+
+* **Tizen Interface Definition Language (TIDL) and RPC-port**: TIDL enables the service applications to communicate with other service application or service daemon and the RPC-port provides application Inter process Communication (IPC) mechanism.
+Applications can expose RPC style service interface to other applications with TIDL.
+
+* **Core Open source Upgrade**: In Tizen 5.0, the following core open source components are upgraded:
+  - Blink M56 which is the core web engine in Tizen is upgraded to M63.
+    - Enhancement to follow W3C Standards such as Animated PNG, Full featured Indexed DB, CSS grid layout, web USB, MSE, EME, and so on.
+    - Enhancement in standard web features WebRTC, and ARIA 1.1
+    - Enhancement in web performance such as V8 : ES6 performance enhancement, Web Assembly default enabling.
+  - GStreamer 1.6.1 is upgraded to 1.12.2.
+  - Pulseaudio 5.0 is upgraded to 11.1.
+  - EFL 1.16 is upgraded to 1.20
+  - Wayland is upgraded to 1.15.0 version.
+  - .NET Core(Runtime) is upgraded
+    - Coreclr 2.1.1 is upgraded to 2.1.4.
+    - Corefx 2.1.1 is upgraded to 2.1.4.
+  - Xamarin.Froms is upgraded to 3.2.0. and so on.
```

Comment:
@jhchoilee 

Slight punctuation additions, spell mistake: 
Copy paste the below sentences at their appropriate places. 

   - Enhancement in standard web features WebRTC and ARIA 1.1.

   - Enhancement in web performance such as V8 : ES6 performance enhancement and Web Assembly default enabling.
- EFL 1.16 is upgraded to 1.20,

- .NET Core(Runtime) is upgraded.


- Xamarin.Forms is upgraded to 3.2.0, and so on.

Reply:
ok